### PR TITLE
add one more cleanup target

### PR DESCRIPTION
**Pull request by BillyWooo**
very tiny PR to detach zombienet to run in background.

### CODE REVIEW

File: parachain/scripts/launch-network.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ fi
 print_divider
 
 echo "launching zombienet network (in background), dir = $ZOMBIENET_DIR ..."
-$ZOMBIENET_BIN -d $ZOMBIENET_DIR -l silent spawn config.toml &
+nohup $ZOMBIENET_BIN -d $ZOMBIENET_DIR -l silent spawn config.toml > /dev/null 2>&1 &
```
Comment (Kailai-Wang) [inline parachain/scripts/launch-network.sh:83]: Hmm why do we need nohup? Isn't this process running in the background already?

Comment (BillyWooo) [reply parachain/scripts/launch-network.sh:83]: Without nohup, there is still dependency on that specific terminal, which logged in. 
It blocks the script to finish completely, even though it looks like everything inside script is already executed successfully.

Comment (Kailai-Wang) [reply parachain/scripts/launch-network.sh:83]: I see, so it's rather for jenkins instead of local setup?

Comment (BillyWooo) [reply parachain/scripts/launch-network.sh:83]: Yes. But it won't affect local setup. I have tested it.

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**